web/book/src/SUMMARY.md

```diff
@@ -57,6 +57,7 @@
 
 - [Bindings](./bindings/README.md)
 
+  - [CLI](./bindings/cli.md)
```

Comment:
@max-sixty Is this the right place for this page?
Or should be "Integrations"?

Reply:
I think it's probably more "integrations" — the bindings are for machines, the integrations are for people